egui/src/widgets/plot/mod.rs

```diff
@@ -314,6 +318,22 @@ impl Plot {
         self.legend_config = Some(legend);
         self
     }
+
+    /// Whether or not to show the background `Rect`.
+    /// Can be useful to disable if the plot is overlaid over existing content.
+    /// Default: `true`.
+    pub fn show_background(mut self, show: bool) -> Self {
+        self.show_background = show;
+        self
+    }
+
+    /// Show the axes.
+    /// Can be useful to disable if the plot is overlaid over an existing grid or content.
+    /// Default: `true`.
+    pub fn show_axes(mut self, show: bool) -> Self {
```

Comment:
It would be nice with a per-axis control here, either with `show_x_axis` + `show_y_axis` or `show_axes(mut self, show: [bool; 2])`. What do you think?